packages/forms/src/index.ts

```diff
@@ -190,12 +190,26 @@ export const addPatterns = (
 export const addPatternToPage = (
   bp: Blueprint,
   pagePatternId: PatternId,
-  pattern: Pattern
+  pattern: Pattern,
+  index?: number
 ): Blueprint => {
   const pagePattern = bp.patterns[pagePatternId] as PagePattern;
   if (pagePattern.type !== 'page') {
     throw new Error('Pattern is not a page.');
   }
+
+  let updatedPagePattern: PatternId[];
+
+  if (index && index !== undefined) {
```

Comment:
Did you intend to exclude `index === 0` from this condition?

```suggestion
  if (index !== undefined) {
```

Reply:
Done.